Code/skyrim_ui/src/app/components/settings/settings.component.ts

```diff
@@ -1,25 +1,24 @@
-import { Component, EventEmitter, HostListener, OnInit, Output } from '@angular/core';
+import { Component, EventEmitter, HostListener, Output } from '@angular/core';
 import { TranslocoService } from '@ngneat/transloco';
 import { ClientService } from 'src/app/services/client.service';
 import { FontSize, SettingService } from 'src/app/services/setting.service';
 import { Sound, SoundService } from '../../services/sound.service';
 
 export enum PartyAnchor {
-  TOP_LEFT,
-  TOP_RIGHT,
-  BOTTOM_RIGHT,
-  BOTTOM_LEFT,
+  TOP_LEFT = 'top_left',
```

Comment:
While I do approve of giving these constant values, are we ok with breaking people's existing settings for the party anchor?

Reply:
Also please make them all upper case.

Reply:
Also I think this enum definition should be in the setting service, with the FontSize enum.